packages/app-mobile/components/screens/Notes/NewNoteButton.tsx

```diff
@@ -85,7 +82,6 @@ const NewNoteButton: React.FC<Props> = _props => {
 				type={ButtonType.Secondary}
 				size={ButtonSize.Larger}
 			>{_('New to-do')}</TextButton>
-			<View style={styles.spacer}/>
```

Comment:
The spacer caused issues with wrapping. Space between the buttons is now managed using the `gap` style property.